kong/dns/client.lua

```diff
@@ -0,0 +1,783 @@
+local cjson = require("cjson.safe")
+local utils = require("kong.dns.utils")
+local mlcache = require("kong.resty.mlcache")
+local resolver = require("resty.dns.resolver")
+
+local now       = ngx.now
+local log       = ngx.log
+local ERR       = ngx.ERR
+local WARN      = ngx.WARN
+local DEBUG     = ngx.DEBUG
+local ALERT     = ngx.ALERT
+local timer_at  = ngx.timer.at
+local worker_id = ngx.worker.id
+
+local pairs           = pairs
+local ipairs          = ipairs
+local math_min        = math.min
+local string_lower    = string.lower
+local table_insert    = table.insert
+local table_isempty   = require("table.isempty")
+
```

Comment:
A couple of missing localizations:
```suggestion
local setmetatable = setmetatable
local tonumber = tonumber

```